lib/obs_github_deployments/cli/commands/obs_command.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ObsGithubDeployments
+  module CLI
+    module Commands
+      class ObsCommand < Dry::CLI::Command
+        protected
+
+        def status_response(status:, reason: nil)
```

Comment:
What about replacing `nil` with an empty string? With `nil` we'll obtain results like

```
$ bin/obs_github_deployments check-lock --repository=$GITHUB_REPOSITORY --token=$GITHUB_TOKEN
=> {"status":"unlocked","reason":null}
```

Reply:
@danidoni I pushed to your branch this little change.

Now I'm going to squash all the commits and this will be ready to merge.